### ISSUE TITLE
"export default from" identifier not supported

### ISSUE THREAD

**Issue by guybedford** [babel/babel#11665]
## Bug Report

**Current behavior**
A clear and concise description of the behavior.

- [REPL](babeljs.io/repl), [Codesandbox](https://codesandbox.io/s/babel-repl-custom-plugin-7s08o?file=/src/index.js), or GitHub Repo helps!

**Input Code**

```js
function from(x) {
    return x === x[Symbol.iterator]() ? Array.from(x) : x;
}
export default from;
```

**Expected behavior**

The above is a valid ES module and will execute in browsers, but throw when parsed by Babylon.

- Babel version(s): [e.g. v6.0.0, v7.0.0-beta.34] 7.10.2

**Possible Solution**

Acorn handles this parsing fine, so it should be a relatively straightforward parser fix.

**Comment by JLHwung** [comment]
Duplicate of #7293

**Comment by JLHwung** [comment]
FYI this will be fixed in #11676.
